### ISSUE TITLE
Corrupted line in src

### ISSUE THREAD

**Issue by jsilves1** [Daniel-KM/Omeka-S-module-AdvancedSearch#16]
After adding AdvancedSearch 3.4.33 to Omeka S, kept getting the following error:

Error: Call to undefined function AdvancedSearch\Form\pù() in 
/omeka-s/modules/AdvancedSearch/src/Form/MainSearchForm.php:1103

Checking the source file MainSerachForm.php found the following near line 1103,

case 'resource_template/o:id':
                $grouped = in_array($filter['type'] ?? '', ['SelectGroup', 'MultiSelectGroup']);
pù($filter);
                return $this->listResourceTemplates($grouped);

Commented out pù($filter); and module appears to work.    Just not sure what pù($filter);   was supposed to be.

**Comment by Daniel-KM** [comment]
Yes, it is a debug code and is fixed in last commit, but the new release is not yet published.

**Comment by Daniel-KM** [comment]
Version 3.4.34 is published.
